README.md

```diff
@@ -60,14 +60,14 @@ choco install ag
 ```
 
 ### Docker
+`ld-find-code-refs` is available as a [docker image](https://hub.docker.com/r/launchdarkly/ld-find-code-refs). The image provides an entrypoint for `ld-find-code-refs`, to which command line arguments may be passed. If using the entrypoint, your git repository to be scanned should be mounted as a volume. Otherwise, you may override the entrypoint and access `ld-find-code-refs` directly from the shell.
 
-`ld-find-code-refs` is available as a [docker image](https://hub.docker.com/r/launchdarkly/ld-find-code-refs). The command line program is installed in the system path of this docker image.
-
-<!-- TODO: update with entrypoint execution when available -->
-
-```shell
+```bash
 docker pull launchdarkly/ld-find-code-refs
-docker run -it launchdarkly/ld-find-code-refs
+docker run \
+  -v /Users/arnold/Documents/projects/launchdarkly/support-service:/repo \
+  launchdarkly/ld-find-code-refs \
```

Comment:
This line should not be necessary if entrypoint is set to ld-find-code-refs

Reply:
How so? `launchdarkly/ld-find-code-refs` is the image name